### ISSUE TITLE
Issue with loading

### ISSUE THREAD

**Issue by Omniscye** [CORRUPTOR2037/MiSide-AssetLoader#2]
Hello,
I have everything installed as required but the mods you included will not even load. It just defaults to the standard game assets.
Is a console supposed to pop up or something? I haven't altered anything and just built it all to test.  Thank you

**Comment by DeathClawDC** [comment]
yes there is supposed to be a console popping up like for any bepinex styled mod loader
